src/Psalm/Internal/CallMap.php

```diff
@@ -11744,7 +11744,7 @@
 'setcookie\'1' => ['bool', 'name'=>'string', 'value='=>'string', 'options='=>'array'],
 'setLeftFill' => ['void', 'red'=>'int', 'green'=>'int', 'blue'=>'int', 'a='=>'int'],
 'setLine' => ['void', 'width'=>'int', 'red'=>'int', 'green'=>'int', 'blue'=>'int', 'a='=>'int'],
-'setlocale' => ['string|false', 'category'=>'int', 'locale'=>'string', '...args='=>'string'],
+'setlocale' => ['string|false', 'category'=>'int', 'locale'=>'string|0|null|', '...args='=>'string'],
```

Comment:
Would you mind removing the trailing `|`?

Reply:
The trailing `|` has been removed